docs/src/main/sphinx/admin/properties-catalog.md

```diff
@@ -0,0 +1,43 @@
+# Catalog management properties
+
+(prop-catalog-management)=
+
+## `catalog.management`
+
+- **Type:** {ref}`prop-type-string`
+- **Allowed values:** `static`, `dynamic`
+- **Default value:** `static`
+
+Trino reads catalog property files and configures available catalogs on server
+startup. When set to `dynamic`, catalog configuration can also be managed using
+[](/sql/create-catalog) and [](/sql/drop-catalog). Creating and dropping
+catalogs using these SQL commands will create new and delete existing catalog
+property files on all nodes.
+
+:::{warning}
+This feature is experimental only. Because of the security implications the
```

Comment:
I dont think the syntax changes and the security implications are necessarily related.

Reply:
I'd leave this, we can always remove this later when we figure out how to handle sensitive values even if there are no syntax changes.

Reply:
ok